db/system_distributed_keyspace.cc

```diff
@@ -525,6 +525,18 @@ system_distributed_keyspace::cdc_get_versioned_streams(db_clock::time_point not_
     co_return result;
 }
 
+future<db_clock::time_point> 
+system_distributed_keyspace::cdc_current_generation_timestamp(context ctx) {
+    auto timestamp_cql = co_await _qp.execute_internal(
+            format("SELECT time FROM {}.{} WHERE key = ? limit 1", NAME, CDC_TIMESTAMPS),
```

Comment:
How important is the performance of this function? If it's important, it could use a prepared statement.

Reply:
Probably not a bad idea for both this and generations query. But I suggest doing as a follow-up. It is not (much) less performant than previous query.